src/main/java/com/exasol/projectkeeper/validators/changelog/DependencySectionFixer.java

```diff
@@ -0,0 +1,85 @@
+package com.exasol.projectkeeper.validators.changelog;
+
+import static com.exasol.projectkeeper.validators.changelog.ChangelogFile.DEPENDENCY_UPDATES_HEADLINE;
+
+import java.io.*;
+import java.nio.file.Path;
+import java.util.*;
+
+import org.apache.maven.model.Build;
+import org.apache.maven.model.Model;
+import org.apache.maven.model.building.ModelBuildingException;
+import org.codehaus.plexus.util.xml.pull.XmlPullParserException;
+
+import com.exasol.errorreporting.ExaError;
+import com.exasol.projectkeeper.validators.changelog.dependencies.*;
+
+/**
+ * This class fixes the dependency section of a {@link ChangelogFile}.
+ */
+class DependencySectionFixer {
+    private final Path projectDirectory;
+    private final Model currentMavenModel;
+
+    /**
+     * Create a new instance of {@link DependencySectionFixer}.
+     * 
+     * @param projectDirectory projects root directory
+     */
+    public DependencySectionFixer(final Path projectDirectory) {
+        this.projectDirectory = projectDirectory;
+        this.currentMavenModel = parseCurrentPomFile(projectDirectory.resolve("pom.xml"));
+    }
+
+    private static Model parseCurrentPomFile(final Path pomFile) {
```

Comment:
I'd separate parsing from the code that fixes something

Reply:
In general, I agree, but in this case, it's about parsing something different than the fixed part. Here we parse the pom file in order to fix the changelog. So we only collect some information for the fix.